lib/ts/recipe/passwordless/index.ts

```diff
@@ -17,8 +17,9 @@ import { UserInput } from "./types";
 
 import Passwordless from "./recipe";
 import PasswordlessAuth from "./passwordlessAuth";
-import { GetRedirectionURLContext, PreAPIHookContext, OnHandleEventContext, RecipeInterface } from "./types";
+import { GetRedirectionURLContext, PreAPIHookContext, OnHandleEventContext } from "./types";
 import SignInUpThemeWrapper from "./components/themes/signInUp";
+import { RecipeInterface } from "supertokens-web-js/recipe/passwordless";
 
 export default class Wrapper {
     static init(config: UserInput) {
```

Comment:
You need to expose all the functions of this recipe?

Please make sure that all the recipes you have implemented so far have done this.

Reply:
Done, also verified in all other recipes